packages/types/lib/driver.ts

```diff
@@ -610,7 +610,6 @@ export interface Core<C extends Constraints, Settings extends StringRecord = Str
   isCommandsQueueEnabled: boolean;
   eventHistory: EventHistory;
   bidiEventSubs: Record<string, string[]>;
-  doesSupportBidi: boolean;
```

Comment:
this property must not be removed as deleting it may break existing drivers, but instead marked as deprecated.